integrations/retail/nemlig.js

```diff
@@ -0,0 +1,137 @@
+import moment from 'moment';
+import request from 'superagent';
+import { CookieAccessInfo } from 'cookiejar';
+
+import { ACTIVITY_TYPE_MEAL } from '../../definitions';
+
+
+/*
+Potential improvements:
+- only refetch items since last fetch.
+*/
+
+// Urls for requests
+const LOGIN_URL = 'https://www.nemlig.com/webapi/login/login';
+const ORDERS_URL = 'https://www.nemlig.com/webapi/order/GetBasicOrderHistory';
+const ORDERS_DETAILS_URL = 'https://www.nemlig.com/webapi/order/GetOrderHistory';
+
+
+// Create an agent that can hold cookies
+const agent = request.agent();
+
+/*
+  Remove cookie injected by Imperva Incapsula that contains character '\u0001'
+  which causes issues on following requests.
+*/
+function removeInvalidCookies() {
```

Comment:
this is unfortunately unsupported in RN. I think we should re-authenticate at each collect()